Templates/JenkinsPipeline/README.md

```diff
@@ -0,0 +1,130 @@
+# Jenkins Multibranch pipeline template
+
+This template provides an Jenkinsfile [MultibranchPipeline](MultibranchPipeline) definition as MultiBranch to setup a Jenkins Multibranch pipeline for applications managed in any Git provider.
```

Comment:
Does "Multibranch pipeline" accurately describe what this template/branching model is trying to convey? Our current recommended branching model (documented [here](https://ibm.github.io/z-devops-acceleration-program/docs/git-branching-model-for-mainframe-dev)) proposes a single `main` branch, and while I guess you could think of feature and epic branches technically making it "multibranch", the name really makes me think of the older Release-based branching model we had with the multiple long-running `main`, `develop`, and release branches.

What if we just changed the naming to be more consistent with how we wrote the other pipeline orchestrator templates (e.g. just use the generically-named "Jenkinsfile" rather than "Multibranch")?

Reply:
@lauren-li thank you for pointing this out. I have updated the documentation and also renamed the template file.

Please see https://github.com/IBM/dbb/pull/245/commits/78ecc810ff0c3718f8ccbd7e37b781b7173e3af6